man/lxqt-policykit-agent.1

```diff
@@ -6,13 +6,13 @@ lxqt-policykit-agent \- Polkit agent for \fBLXQt\fR: the faster and lighter QT D
 [\fICurrently No command line arguments\fR]
 .br
 .SH DESCRIPTION
-This module handles policykit for \fBLXQt\fR. The polkit module, will perform 
+This module handles policykit for \fBLXQt\fR. The polkit module will perform 
 security tast over scale action privilegies.
```

Comment:
I have no idea what this sentence should mean. 'security tast over scale action privilegies'?